src/pages/docs/argo-cd/instances/argo-user.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2025-09-15
+modDate: 2025-09-15
+title: Octopus user in Argo
+description: Limiting Octopus' access in Argo CD
+navOrder: 10
+hideInThisSectionHeader: true
+---
+
+OctopusDeploy fetches application, cluster and log data from your Argo CD Instance. This data is used in the Octopus UI to provide
+a rich integration, and also during step execution to determine which applications/repositories are to be updated.
+
+To request this data, Octopus must authenticate with Argo CD as a user with appropriate permissions.
+
+While a new token could be generated for an existing user, it is highly advised that a new user be created within Argo to
+represent the Octopus interactions.
+
+To do this, the following must be performed:
+1. Create a new user in Argo CD
+2. Create RBAC policies to allow the new user read-access to required resources
+3. Generate an authentication token for the new user
+
+## Create a new User
+To create a new user in Argo, you must update the `argocd-cm` configmap (typically in the argo-cd namespace).
+
+The following shows a configmap with a new user called `octopus` which is able to generate an apiKey, but cannot login
+via Argo's web-ui.
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+name: argocd-cm
+namespace: argocd
+labels:
+app.kubernetes.io/name: argocd-cm
+app.kubernetes.io/part-of: argocd
+data:
+# add an additional local user with apiKey and login capabilities
+#   apiKey - allows generating API keys
+  accounts.octopus: apiKey
+  accounts.octopus.enabled: true
```

Comment:
I think the `true` needs to be quoted - otherwise Lens gives me an error

Reply:
done